variables.tf

```diff
@@ -5,7 +5,7 @@ variable "kms_key_arn" {
 }
 
 variable "rules" {
-  type        = list(map(any))
+  type        = any
```

Comment:
Why is this set to `any` instead of `list(any)`

Reply:
yep i agree it should be `list(any)`. Will change